### ISSUE TITLE
Create a User model

### ISSUE THREAD

**Issue by Lauren-Baca** [Grace-Shopper-E/grace-shopper#17]
Should have at least username and password attributes

A user is associated with one cart

**Comment by christine-zheng** [comment]
User Model for database:

username: not null, not empty and unique,
password: not null and not empty
email: make sure it is unique
userType: ENUM (customer, admin)

**Comment by christine-zheng** [comment]
yep, the boilerplate already has the hook to hash password before adding it to the database, so thats set up for us at least. :D
